test/largescale/solvers/advanced/proximal_operator_slow_test.py

```diff
@@ -31,9 +31,11 @@
 from odl.solvers.advanced.proximal_operators import (
     proximal_l1, proximal_cconj_l1,
     proximal_l2, proximal_cconj_l2,
-    proximal_l2_squared, proximal_cconj_l2_squared)
-from odl.util.testutils import noise_element
+    proximal_l2_squared, proximal_cconj_l2_squared,
+    proximal_cconj_kl, proximal_cconj_kl_version_two)
+from odl.util.testutils import (noise_element, all_almost_equal)
 
+from scipy.special import lambertw
```

Comment:
simply `import scipy` and then do `scipy.special.lambertw` below. More clear where it comes form.